docs/framework/litellm.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Using Composio With LiteLLM"
+sidebarTitle: "LiteLLM"
+icon: "people-group"
+description: "Integrate Composio with LiteLLM and seamlessly interact with external apps"
+---
+
+## Star A Repository on Github
+In this example, we will use LiteLLM to star a repository on Github using Composio Tools
+
+<Steps>
+<Step title="Install Packages">
+```bash Python
```

Comment:
The code block specifies ```bash Python``` which is incorrect. It should be either ```bash``` for shell commands or ```python``` for Python code. In this case, since it's a pip install command, use ```bash```.